src/components/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import aboutIcon from "../assets/About.svg"
+import teamIcon from "../assets/Team.svg"
+import githubIcon from "../assets/GitHub.svg"
+import whatsNewIcon from "../assets/WhatsNew.svg"
+import discordIcon from "../assets/Discord.svg"
+
+type Props = {};
+
+const Footer = (props: Props) =>{
+  const flexBetween = "flex justify-evenly";
+  return <nav> 
+
+          <div className = "flex justify-between flex-row fixed inset-x-0 bottom-12">

```

Comment:
I checked how the footer integrates with the content component and found that it floats above it. I poked around and noticed that the `"fixed"` tag seems to be the culprit. Removing it seemed to solve the issue.

Reply:
Once you're able to pull changes from master, check the padding around the footer, as well as the sizing in comparison to the other components. Let me know if my content seems too small--I can increase the sizing if needed.

Reply:
Yes once I merged the changes I saw the issue, I fixed it by using a static position instead of fixed. In regards to icon and font sizes, I think its something we need to go over together so we match our design to that of the Figmas.